test/invalid/json/duplicate-escaped.json

```diff
@@ -0,0 +1,4 @@
+{
+    "message": "Got an error during or right after parsing `interface Iroha`: The name \"Iroha\" of type \"interface\" is already seen",
```

Comment:
nit: s/is/was

Reply:
That means is->was, right? What does `s/` part mean?

Reply:
Ah sorry, bad habit `s/` meaning replace... something we do at the w3c :(

Reply:
(I think "s/" is a vim thing)

Reply:
Oh, TIL an interesting w3c syntax 😁